coord/src/rmr/rmr.c

```diff
@@ -512,6 +512,7 @@ typedef struct MRIteratorCtx {
   void *privdata;
   MRIteratorCallback cb;
   int pending;
+  int inProcess;
```

Comment:
Maybe call it pending cursor requests? It was not trivial for me to understand what it is (and maybe I still got it wrong :))

Reply:
`pending` was “not done”. `inProcess` is “actually executing right now on the shards”. I’ll think of another name (for both?)